app/mailers/candidate_mailer.rb

```diff
@@ -434,6 +436,33 @@ def application_deadline_has_passed(application_form)
     )
   end
 
+  def respond_to_offer_before_deadline(application_form)
+    @decline_by_default_deadline = CycleTimetable.decline_by_default_date.to_fs(:govuk_date)
+    year = application_form.recruitment_cycle_year
+    @this_academic_year = CycleTimetable.cycle_year_range(year)
+    @next_academic_year = CycleTimetable.cycle_year_range(year + 1)
+    @apply_reopens_date = CycleTimetable.apply_reopens.to_fs(:govuk_date)
+    email_for_candidate(
+      application_form,
+      subject: I18n.t!(
+        'candidate_mailer.respond_to_offer_before_deadline.subject',
+        decline_by_default_date: @decline_by_default_deadline,
+      ),
+    )
+  end
+
+  def reject_by_default_explainer(application_form)
+    year = application_form.recruitment_cycle_year
+    @this_academic_year = CycleTimetable.cycle_year_range(year)
+    @next_academic_year = CycleTimetable.cycle_year_range(year + 1)
+    @apply_reopens_date = CycleTimetable.apply_reopens.to_fs(:govuk_date)
+
+    email_for_candidate(
+      application_form,
+      subject: I18n.t!('candidate_mailer.reject_by_default_explainer.subject'),
```

Comment:
I didn't realise you could do this `!` 👍🏼 

We should probably be using these in most of our translation calls.

Reply:
I suppose it would raise an translation exceptions in production, and we might need to decide how we want that to be handled. Like do we want to catch and it send it to Sentry, but still render the page with some generic text? Or do we want the page to error? But good to do it here because we don't want to send a bunch of emails where we might look like spam.

Reply:
I think generally if we can, we should catch it and send it to Sentry, but still render the page (with the translation missing). 

It feels wrong when a missing translation literally cause a whole page to 500